benchmark-service/benchmark/config.py

```diff
@@ -4,7 +4,7 @@
 
 HAIL_BENCHMARK_BUCKET_NAME = os.environ['HAIL_BENCHMARK_BUCKET_NAME']
 
-INSTANCE_ID = os.environ.get('INSTANCE_ID')
+INSTANCE_ID = os.environ.get('INSTANCE_ID', 'R8kZlZhmqGJk')
```

Comment:
So what I think we actually want is to modify the deployment.yaml file to return this ID as the environment variable if the scope is dev.